File: python/Iniciante/URI1045.py
a, b, c = map(float, input().split())

if b > a and b > c:
  start = a
  a = b
  b = start
elif c > a:
  start = a
  a = c
  c = start

if a >= (b + c):
  print("NAO FORMA TRIANGULO")
else:
  if (a * a) == (b * b) + (c * c):
    print("TRIANGULO RETANGULO")
  elif (a * a) > (b * b) + (c * c):
    print("TRIANGULO OBTUSANGULO")
  elif (a * a) < (b * b) * (c * c):
    print("TRIANGULO ACUTANGULO")

  if a == b and a == c:
    print("TRIANGULO EQUILATERO")
  elif ((a == b and a != c) or (a == c and a != b) or (b == c and a != b)):
    print("TRIANGULO ISOSCELES")

File: java/iniciante/URI1045.java
package URI;

import java.util.Scanner;

public class URI1045 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double A = scanner.nextDouble();
        double B = scanner.nextDouble();
        double C = scanner.nextDouble();
        double start;

        if (B > A && B > C) {
            start = A;
            A = B;
            B = start;
        } else if (C > A)  {
            start = A;
            A = C;
            C = start;
        }

        if (A >= (B + C)) {
            System.out.println("NAO FORMA TRIANGULO");
        } else {
            if ((A * A) == (B * B) + (C * C) ) {
                System.out.println("TRIANGULO RETANGULO");
            } else if ((A * A) > (B * B) + (C * C)) {
                System.out.println("TRIANGULO OBTUSANGULO");
            } else if ((A * A) < (B * B) + (C * C)) {
                System.out.println("TRIANGULO ACUTANGULO");
            }
            if (A == B && A == C) {
                System.out.println("TRIANGULO EQUILATERO");
            } else if ((A == B && A != C) || (A == C && A != B) || (B == C && A != B)) {
                System.out.println("TRIANGULO ISOSCELES");
            }
        }
    }
}
